src/support/slack/commands/run-scrape.js

```diff
@@ -85,18 +85,23 @@ function RunScrapeCommand(context) {
       const urls = topPages.map((page) => ({ url: page.getURL() }));
       await say(`:white_check_mark: Found top pages for site \`${baseURL}\`, total ${topPages.length} pages.`);
 
-      const jobId = site.getId();
-      await triggerScraperRun(
-        jobId,
-        urls,
-        slackContext,
-        context,
-      );
-      await say(`:adobe-run: Triggered scrape run for site \`${baseURL}\` — Total ${urls.length} URLs`);
-
-      const message = `:white_check_mark: Completed triggering scrape runs for site \`${baseURL}\` — Total URLs: ${urls.length}`;
-
-      await say(message);
+      const half = Math.ceil(urls.length / 2);
```

Comment:
I think we should not depend on the count of the urls, lets say in future we start saving more than 200 pages. In that case we would start scraping more than 100 pages in which case the timeout issue can arise again.
Also, if the total top pages is not a large number, IIRC we have 17 for sunstar, we would divide that into 8, 7 and send 2 scrape messages which would be unnecessary.
To avoid these issues, lets process batches of a fixed number of urls, 50 maybe.